src/Workspaces/CSharp/Portable/Extensions/ContextQuery/SyntaxTreeExtensions.cs

```diff
@@ -1039,6 +1040,24 @@ public static bool IsDefaultExpressionContext(this SyntaxTree syntaxTree, int po
                 return true;
             }
 
+            if (allowedModifiersBefore != null &&
+                allowedModifiersBefore.Contains(token.Kind()) &&
+                token.Parent.GetParent().IsDelegateOrConstructorOrLocalFunctionOrMethodParameterList())
+            {
+                if (allowableIndex.HasValue)
```

Comment:
For that case, yes. Am I missing something?

---
In reply to: [120768306](https://github.com/dotnet/roslyn/pull/19703#discussion_r120768306) [](ancestors = 120768306)